packages/okta-vue/test/e2e/harness/package.json

```diff
@@ -6,14 +6,16 @@
   "private": true,
   "scripts": {
     "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
-    "start": "npm run dev",
+    "start": "npm run env && npm run dev",
+    "env": "node ./scripts/prebuild.js",
     "e2e": "node test/e2e/runner.js",
-    "test": "npm run e2e",
+    "test": "npm run env && npm run e2e",
     "lint": "eslint --ext .js,.vue src test/e2e/specs",
     "build": "node build/build.js"
   },
   "dependencies": {
     "@okta/okta-auth-js": "^1.11.0",
+    "chromedriver": "^2.33.0",
```

Comment:
Looks like this got changed back. Any reason why?

Reply:
My bad. Will put it back to where it belongs.